app/src/components/Verto/StakeStepper.vue

```diff
@@ -481,14 +497,14 @@ export default {
       this.checkAmount()
     },
     checkAmount () {
-      let stake_per = Math.round((0.01 + (0.001 * this.stakePeriod)) * 1000) / 1000
+      let stake_per = (1 + this.stakePeriod / 10.0) / 100
```

Comment:
I don't think this is right. 
Prove me wrong!

Reply:
I got it from the contract... I saw that the rewards value were not equal to the expected total calculated in the UI

` asset calculate_subsidy(asset quantity, uint16_t periods_num) 
   {
      check_quantity(quantity);
      check(periods_num <= 10, "max allowed periods num is 10 stake periods");
      double subsidy_part = ( 1 + periods_num / 10.0 ) / 100;
      asset subsidy = asset(0, symbol(TOKEN_SYMBOL, SYMBOL_PRE_DIGIT));
      subsidy.amount = quantity.amount * subsidy_part;
      return subsidy; 
   }`